argo/0.9.1/artifacthub-pkg.yml

```diff
@@ -0,0 +1,23 @@
+version: 0.9.1
+name: argo-service
+displayName: Argo Service
+createdAt: '2022-01-24T12:31:01Z'
+description: Keptn service for Argo CD - Declarative GitOps CD for Kubernetes.
+logoURL: https://raw.githubusercontent.com/cncf/artwork/master/projects/argo/stacked/color/argo-stacked-color.svg
+digest: '2022-01-24T12:31:01Z'
+license: Apache-2.0
+homeURL: https://keptn.sh/docs/integrations/
+keywords:
+- keptn
+- argo
+- deployment
+- contrib
+links:
+- name: Source
+  url: https://github.com/keptn-contrib/argo-service
+recommendations:
+- url: https://artifacthub.io/packages/helm/keptn/keptn
+annotations:
+  keptn/kind: deployment
+  keptn/org: contrib
+  keptn/version: 0.8.4-0.8.7
```

Comment:
Seems like this is copied from `0.8.4` and not from `0.9.0`. Will look into why this happens.

Reply:
Fixed in #66